homeassistant/components/yeelightsunflower/light.py

```diff
@@ -45,13 +44,19 @@ def __init__(self, light):
         self._available = light.available
         self._brightness = light.brightness
         self._is_on = light.is_on
-        self._hs_color = light.rgb_color
+        self._rgb_color = light.rgb_color
+        self._unique_id = self.name
```

Comment:
We don't need to include `sunflower_` in the unique_id. The entity registry is aware of both the integration domain and the platform domain.

What kind of id is the `zid`?

Reply:
It's a 4-digit HEX like `6553` or `287B`. 
What would you like the `unique_id` to be instead?

Reply:
If the `zid` is unique and stable, just use that. :+1:

Reply:
OK. It is unique and stable, so I will. 
I had `sunflower_` in there to be more descriptive. `light.6553` doesn't give as much information, but I guess a user probably knows where their light entities came from :)